src/notification.rs

```diff
@@ -296,6 +308,21 @@ impl WMIConnection {
     }
 }
 
+/// Ongoing notification subscription.
+///
+/// To cancel and stop the notification, simply drop this object.
+pub struct Notification<'a> {
+    connection: &'a WMIConnection,
+    sink: ClassAllocation<QuerySink>,
+}
+
+impl Drop for Notification<'_> {
+    fn drop(&mut self) {
+        let sink = IWbemObjectSink::from(&**self.sink);
+        unsafe { (*self.connection.svc()).CancelAsyncCall(sink.get_abi().as_ptr() as *mut _) };
```

Comment:
```suggestion
        unsafe { (*self.connection.svc()).CancelAsyncCall(sink.get_abi().as_ptr().cast_mut()) };
```
Probably works

Reply:
Else `.cast().cast_mut()`, in whatever order

Reply:
This is symmetric to what is done in many other places in the crate, and is going to be removed in the windows-rs transition anyway, so i'll keep it this way